packages/babel-helpers/src/helpers.js

```diff
@@ -75,9 +75,9 @@ helpers.asyncToGenerator = template(`
             resolve(value);
           } else {
             return Promise.resolve(value).then(function (value) {
```

Comment:
is this `return` necessary either? was looking at https://github.com/leebyron/async-to-gen/commit/804c72d68c1457fde56f237324d4ca7585921f40 which is basically the same helper @leebyron